Cargo.toml

```diff
@@ -36,9 +36,10 @@ name = "connected_to_journal"
 harness = false
 
 [package.metadata.release]
-sign-commit = true
 disable-publish = true
 disable-push = true
-pre-release-commit-message = "cargo: libsystemd release {{version}}"
 post-release-commit-message = "cargo: development version bump"
+pre-release-commit-message = "cargo: libsystemd release {{version}}"
+sign-commit = true
+sign-tag = true
```

Comment:
I guess this means I'll need to get myself a GPG key?  I must admit that so far I didn't bother, but if you prefer it this way I can do that.

Will take a bit, as I'm not quite familiar with GPG so I'll need to through some tutorials, but I don't really have much time for the next two weeks :slightly_smiling_face: